### PR TITLE
Log appset version during controller startup

### PR DESCRIPTION
**Pull request by chetan-rns**
Display the version of applicationset and git commit ID during controller startup log

Closes: #169

### CODE REVIEW

File: common/version.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package common
+
+// version information populated during build time
+var (
+	version   = "v0.2.0" // value from VERSION file
```
Comment (jgwest) [inline common/version.go:5]: Can this be `version = "" // value from VERSION file`, or is there an advantage to having `v0.2.0` hardcoded in this file?

**Review by jgwest — CHANGES_REQUESTED** [review]
Looks great! Just one question re: version string, then good to go.

File: common/version.go (hunk 1 above)
Comment (chetan-rns) [reply common/version.go:5]: I thought we needed a default value here, but looks we can always get it from the VERSION file. Will change it to an empty string.

**Review by jgwest — APPROVED** [review]
Excellent, thanks again @chetan-rns!

**Pull request merged**